### PR TITLE
SW-2037 / SW-2448 Various button style updates

### PR DESCRIPTION
**Pull request by alexTerraformation**
Change some icon buttons to 'ghost' priority.  Change 'Create' to 'Save' for accessions

### CODE REVIEW

File: src/components/NurseryWithdrawals/NurseryWithdrawals.tsx [hunk 1]
```diff
@@ -251,9 +242,13 @@ export default function NurseryWithdrawals(): JSX.Element {
                 value={searchValue}
                 onChange={(value) => setSearchValue(value as string)}
               />
-              <IconButton onClick={handleFilterClick} size='medium' className={classes.iconContainer}>
```
Comment (karthikbtf) [inline src/components/NurseryWithdrawals/NurseryWithdrawals.tsx:254]: remove the unused import.. and in other files (if applicable)?

File: src/components/NurseryWithdrawals/NurseryWithdrawals.tsx [hunk 2]
```diff
@@ -251,9 +242,13 @@ export default function NurseryWithdrawals(): JSX.Element {
...
-                <Icon name='filter' className={classes.icon} size='medium' />
-              </IconButton>
+              <Button
```
Comment (karthikbtf) [inline src/components/NurseryWithdrawals/NurseryWithdrawals.tsx:245]: nice change!

File: src/components/NurseryWithdrawals/NurseryWithdrawals.tsx [hunk 3]
```diff
@@ -251,9 +242,13 @@ export default function NurseryWithdrawals(): JSX.Element {
...
+                id='filterNurseryWithdrawal'
+                onClick={() => handleFilterClick}
```
Comment (karthikbtf) [inline src/components/NurseryWithdrawals/NurseryWithdrawals.tsx:247]: I think the onClicks are broken.. you can change this to
`onClick={handleFilterclick}` or `onClick={() => handleFilterClick()}`

**Review by karthikbtf — COMMENTED** [review]
Nice change for consistency 👍 
A few comments...

File: src/components/NurseryWithdrawals/NurseryWithdrawals.tsx (hunk 3 above)
Comment (alexTerraformation) [reply src/components/NurseryWithdrawals/NurseryWithdrawals.tsx:246]: Oops, missed this thanks!

**Review by karthikbtf — APPROVED** [review]
lgtm 👍

**Pull request merged**